### DISCUSSION TITLE
npm publish throws code E400 after entering OTP token in the browser

### DISCUSSION THREAD

**Discussion by ad-on-is** [community/community#124854] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

- Run "npm publish" in CLI
- `Press ENTER to open in the browser...`
- Go to browser and enter OTP token
- Result in CLI: `npm error 400 Bad Request - PUT https://registry.npmjs.org/xxx - child "otp" fails because ["otp" with value "[object Object]" fails to match the required pattern: /^\d+$/, "otp" length must be 64 characters long]`

**Comment by mikeseese** [comment]
@ad-on-is This is the GitHub feedback community. The correct place to file this bug is on NPM's repository. I also have the issue and posted one here: https://github.com/npm/cli/issues/7547

I was able to work around this bug by downgrading NPM: `npm install --global npm@10.7.0`

**Comment by ad-on-is** [comment]
Lol... I went to the npm repo, submitted an issue there, but somehow it was created here.
